office365-x64/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1 @@
+start-process "C:\Program Files\Common Files\Microsoft Shared\ClickToRun\OfficeClickToRun.exe" -arg "scenario=install scenariosubtype=ARP sourcetype=None productstoremove=O365ProPlusRetail.16_en-us_x-none culture=en-us version.16=16.0 DisplayLevel=False" -wait
```

Comment:
You might want Start-ChocolateyProcessAsAdmin here but this is fine for this now